### DISCUSSION TITLE
刚刚的迪奥娜生日消息是不是重复推送了很多遍？

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#583] [General]
我这边被重复推了三遍
<details>
<summary></summary>

![image](https://user-images.githubusercontent.com/9006264/149870635-aa69c171-c577-405d-b160-e8265cf43a9a.jpeg)
</details>

**Comment by Arondight** [comment]
![image](https://user-images.githubusercontent.com/13673702/149871640-f75fd047-d5d4-48e0-ba55-d990c4a48788.png)

我这里正常

**Reply by mark9804** [reply]
等我回家看看 log 吧，早上可莉消息推送失败了，之后开始重复推送（不知道有没有关联），现在人在外面也连不上内网树莓派

早上的失败记录我开在自己 issue 备查了

https://github.com/Mark9804/Adachi-BOT/issues/75

**Comment by mark9804** [comment]
爷等了半年（物理意义半年）今天终于换宽带了，米游社访问快得飞起，索尼大法好啊
<img width="736" alt="image" src="https://user-images.githubusercontent.com/9006264/155968785-a01cd764-326a-46de-b0cd-2c0704045a82.png">
